moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -1,5 +1,29 @@
 import { FiltersCommonOperators } from "../../../components/GridTable/FiltersCommonOperators";
 
+/**
+ * Array of operators that should be autocompleted
+ * @constant
+ * @type {array}
+ * @default
+ */
+export const AUTOCOMPLETE_OPERATORS = [
+  "string_equals_case_sensitive",
+  "string_does_not_equal_case_sensitive",
+];
+
+/**
+ * Array of operators that do not require a search value Ex. "is blank"
+ * @constant
+ * @type {array}
+ * @default
+ */
+export const OPERATORS_WITHOUT_SEARCH_VALUES = [
+  "string_is_null",
+  "string_is_not_null",
+  "string_is_not_null_special_case",
+  "string_is_null_special_case",
+];
```

Comment:
little stuff like this goes a long way toward making the mechanics of all this easier to follow 🙌

Reply:
thanks - that was my hope! forgot to mention that this fixes a bug as well. The "special case" columns that used to return "None" were preventing additional filters from being added. Thinking that we will need to update this when we fix the broken `is_blank` filters config.